web/template/investigation_off_hold.gohtml

```diff
@@ -0,0 +1,47 @@
+{{ template "page" . }}
+
+{{ define "title" }}{{ if .Error.Any }}Error: {{ end }}Investigation off hold{{ end }}
```

Comment:
I think these titles are kind of confusing because it sounds like it is describing the current state of the hold (but you don't have to change them). But it did make me wonder whether the on/off hold pages should actually be one page that switches between depending on what the state is?

Reply:
Yeah those titles are confusing, i'll ask Katie and maybe get a ticket created to change them. I agree it would be better to have 1 template with switches based off the `isOnHold` value, however it might be better to do this in a separate ticket as this work is blocking the edit investigation ticket, and its just opened a can of worms in my mind...I think there needs to be more defensive coding, eg checking the state and preventing putting on/off hold if the state is already that which the user is intending. Im thinking ahead to when we don't have angular JS and these actions may have their own page, and the `id` URL param could be changed, or a user could go back to a previously visited page with a different ID, we should check the state and maybe displaying a message to say "This investigation is already on hold" etc (Unlikely scenarios I know, but the fact that you can currently put an investigation on hold which is already on hold etc needs sorting)

Reply:
Yeah I was thinking more that than just the template with my comment, so you'd have `/investigation-hold?id=X` (or whatever) and it would display the correct thing.